src/pages/workspaces/workspace/Data.js

```diff
@@ -1076,7 +1052,7 @@ const WorkspaceData = _.flow(
               setSelectedData({ type: workspaceDataTypes.entities, entityType: tableName })
             })
           })],
-          [workspaceDataTypes.wds, () => wdsDataTableProvider && wdsProxyUrl && wdsSchema && h(WDSContent, {
+          [workspaceDataTypes.wds, async () => wdsDataTableProvider && (await wdsDataTableProvider.proxyUrlPromise) && wdsSchema && h(WDSContent, {
```

Comment:
The return value of this function is going to rendered by React. It shouldn't return a Promise.